subgraph.yaml

```
@@ -125,7 +125,61 @@ dataSources:
 
   # MARKETS
   - kind: ethereum/contract
-    name: Market
+    name: HL Mock DIA 1
```

Let's make it clear that this is a Market and not a Mock DIA (whatever DIA is)
```suggestion
    name: DIA Market
```